android/library/src/main/java/io/logto/android/auth/browser/BrowserLoginFlow.kt

```diff
@@ -17,20 +17,20 @@ import io.logto.android.utils.UrlUtil
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 
-object BrowserFlow {
+object BrowserLoginFlow {
 
-    private var authConfig: BrowserAuthConfig? = null
+    private var loginConfig: BrowserLoginConfig? = null
 
     fun init(
         logtoConfig: LogtoConfig,
         authenticationCallback: AuthenticationCallback
-    ): BrowserFlow {
+    ): BrowserLoginFlow {
         resetFlow()
 
         val codeVerifier = Util.generateCodeVerifier()
         val authUrl = generateAuthUrl(logtoConfig, codeVerifier)
 
-        authConfig = BrowserAuthConfig(
+        loginConfig = BrowserLoginConfig(
```

Comment:
currently sign in/out config can be the same stuff, you can keep `AuthConfig` for now